scripts/src/osd/modules.lua

```diff
@@ -13,6 +13,10 @@ function string.starts(String,Start)
 	return string.sub(String,1,string.len(Start))==Start
 end
 
+function string.ends(String,EndPart)
+	return string.sub(String,string.len(String)-string.len(EndPart)+1)==EndPart
+end
+
```

Comment:
It’s cleaner to implement this as `(EndPart == "") or String:sub(-#EndPart) == EndPart`

Reply:
Done